packages/pigeon/test/swift_generator_test.dart

```diff
@@ -797,14 +797,8 @@ void main() {
     final String code = sink.toString();
     expect(code, contains('func add(x: Int64, y: Int64) throws -> Int64'));
     expect(code, contains('let args = message as! [Any]'));
-    expect(
-        code,
-        contains(
-            'let xArg = (args[0] is Int) ? Int64(args[0] as! Int) : args[0] as! Int64'));
-    expect(
-        code,
-        contains(
-            'let yArg = (args[1] is Int) ? Int64(args[1] as! Int) : args[1] as! Int64'));
+    expect(code, contains('let xArg: Int64 = args[0] as! Int64'));
+    expect(code, contains('let yArg: Int64 = args[1] as! Int64'));
```

Comment:
The more we learn about the differing and surprising behavior of Swift casting, the more this makes me nervous, because (like NSNull->Foo? casting) it doesn't actually seem like this *should* work. Is this cross-int-type-casting behavior actually documented anywhere? If not, I don't think we should rely on it.